corpus/convert.py

```diff
@@ -109,25 +114,33 @@ def run(self):
         c.dump(self.out_corpus.get_path())
 
 
-class CorpusToStmJob(Job):
+class CorpusToStmJob(_CorpusToStmJob):
+    pass
+
+
+class CorpusToStmJobV2(Job):
```

Comment:
I don't think we need to deprecate the old job. you can add the new parameters via sis_hash_exclude.

we don't even have to change the default `exclude_non_speech` and `remove_punctuation` from `True` to `False` because with empty tuples the for-loops below will do exactly nothing (i.e. default behavior of the original job)

The only question is, if we silently want to implement the fix_whitespace option. I think this is harmless and definitively worth it to avoid a V2 Job.

Reply:
If possible I would also avoid the V2 if were are not really having two different jobs with different behavior.

We certainly can write the hash function in a way to replicate old hashing.